metricbeat/module/kubernetes/state_deployment/_meta/test/ksm.v2.4.2.plain.expected

```diff
@@ -59,6 +59,10 @@
 		"MetricSetFields": {
 			"name": "coredns",
 			"paused": false,
+			"status": {
```

Comment:
nit: fix indentation pls.

Same for the all the following files that need it.

Reply:
fixed